src/test/java/com/hubsante/model/validator/TechnicalValidatorTest.java

```diff
@@ -0,0 +1,225 @@
+/**
+ * Copyright © 2023-2024 Agence du Numerique en Sante (ANS)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hubsante.model.validator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static com.hubsante.model.config.Constants.TECHNICAL_SCHEMA;
+import static com.hubsante.model.config.Constants.TECHNICAL_XSD;
+import static com.hubsante.model.utils.TestFileUtils.getMessageByFileName;
+
+public class TechnicalValidatorTest extends AbstractValidatorTest {
+
+    @Test
+    @DisplayName("Valid technical messages should pass validation")
+    public void jsonTechnicalValidationPasses() throws Exception {
+        validator.validateJSON(getMessageByFileName("TECHNICAL/complete.json"), TECHNICAL_SCHEMA);
+        validator.validateJSON(getMessageByFileName("TECHNICAL/array-deserialization.json"), TECHNICAL_SCHEMA);
+        validator.validateJSON(getMessageByFileName("TECHNICAL/nomenclature-test.json"), TECHNICAL_SCHEMA);
+        validator.validateJSON(getMessageByFileName("TECHNICAL/regex-validation.json"), TECHNICAL_SCHEMA);
+
+        validator.validateXML(getMessageByFileName("TECHNICAL/complete.xml"), TECHNICAL_XSD);
+        validator.validateXML(getMessageByFileName("TECHNICAL/array-deserialization.xml"), TECHNICAL_XSD);
+        validator.validateXML(getMessageByFileName("TECHNICAL/nomenclature-test.xml"), TECHNICAL_XSD);
+        validator.validateXML(getMessageByFileName("TECHNICAL/regex-validation.xml"), TECHNICAL_XSD);
+    }
+
+    @Test
+    @DisplayName("Messages with values that do not pass regex validation should not pass validation")
+    public void technicalRegexValidationFails() throws IOException {
+        String[] expectedJsonErrors = {
+            "Could not validate message against schema : errors occurred. ",
+            "Issues found on the envelope: ",
+            " - $.phoneNumberField: does not match the regex pattern \\+?[0-9]{2,14}",
+            " - $.datetimeField: 2022-09-27 08:25:54 Z02:00 is an invalid date-time",
+            " - $.datetimeField: does not match the regex pattern \\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}[\\-+]\\d{2}:\\d{2}",
+            " - $.dateField: does not match the regex pattern \\d{4}\\-(0[1-9]|1[012])\\-(0[1-9]|[12][0-9]|3[01])",
+            " - $.emailField: does not match the regex pattern [\\w\\-\\.]+@([\\w\\-]+\\.)+[\\w\\-]{2,4}"
```

Comment:
Est-ce qu'on ne devrait pas juste checker "- $.datetimeField: does not match the regex pattern" pour pouvoir changer les regex si besoin ? Après peut-être pas...

Reply:
Ca me parait un peu plus secure de checker les messages d'erreur en full; meme si ca implique la modification potentiellement un peu fastidieuse des TUs a chaque changement dans le mdd, ca nous permet d'etre completement certains du sortie de validateur.
En plus, j'imagine pas que les regex vont souvent changer.